src/components/Employee/Taxes/FederalForm.tsx

```diff
@@ -32,17 +32,14 @@ export function FederalForm() {
 
   return (
     <>
-      <Select
-        control={control}
+      <SelectField
         name="filingStatus"
         label={t('federalFilingStatus1c')}
         placeholder={t('federalFillingStatusPlaceholder')}
-        items={filingStatusCategories}
+        options={filingStatusCategories.map(cat => ({ value: cat.id, label: cat.name }))}
```

Comment:
Same feedback, we can probably just update the `filingStatusCategories` constant to have `value` and `label` instead of needing to map

Reply:
Updated